pkgs/tools/virtualization/mkosi/default.nix

```
@@ -85,7 +85,7 @@ buildPythonApplication rec {
     [
       (replaceVars ./0001-Use-wrapped-binaries-instead-of-Python-interpreter.patch {
         UKIFY = "${systemdForMkosi}/lib/systemd/ukify";
-        PYTHON_PEFILE = "${python3pefile}/bin/python3.12";
+        PYTHON_PEFILE = "${pythonWithPefile}/bin/python3.12";
```

```suggestion
        PYTHON_PEFILE = lib.getExe pythonWithPefile;
```
Would be accurate for different default Pythov rersion